manager/state/raft/raft.go

```diff
@@ -100,6 +100,9 @@ type Node struct {
 	// waitProp waits for all the proposals to be terminated before
 	// shutting down the node.
 	waitProp sync.WaitGroup
+	// used to campaign once on node restore if we are the only
+	// registered member in the cluster memberlist.
+	once sync.Once
```

Comment:
`sync.Once` is generally used where thread safety is required. Here, a simple boolean would work. I'm okay with using `sync.Once` even though it's overkill, but the comment should probably note that we're not relying on it for thread safety.